jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/FormsSizingTests.java

```diff
@@ -86,15 +86,17 @@ public void isValidationTest() {
         formsSizing.smallSelect.is()
                 .displayed()
                 .selected("Small select");
-        formsSizing.largeTextField.is()
-                .enabled()
-                .placeholder(placeholderLarge);
+        formsSizing.largeTextField
+                .placeholder()
```

Comment:
missed is (whithout is it just a method that returns true or false without actual asserts)